fab/fabfile.py

```diff
@@ -436,6 +436,22 @@ def apply_patch(patchfile=None):
     silent_services_restart(use_current_release=True)
 
 
+@task
+def reverse_patch(patchfile=None):
+    """
+    Used to reverse a git patch created via `git format-patch`. Usage:
+
+        fab <env> reverse_patch:patchfile=/path/to/patch
+
+    Note: Only use this when absolutely necessary.
```

Comment:
Might be worth expounding on why? I don't know the reasons, but I'm guessing because it might work and it might not, and if it fails it's probably going to be a bad time (possibly involving service downtime?).